apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverview.tsx

```diff
@@ -152,8 +157,40 @@ export const SignedVerdictOverview: React.FC = () => {
                 {
                   // Custody restrictions
                   workingCase.decision === CaseDecision.ACCEPTING &&
-                    workingCase.custodyRestrictions?.map(
-                      (custodyRestriction, index) => (
+                    workingCase.custodyRestrictions
+                      ?.filter((restriction) =>
+                        [
+                          CaseCustodyRestrictions.ISOLATION,
+                          CaseCustodyRestrictions.VISITAION,
+                          CaseCustodyRestrictions.COMMUNICATION,
+                          CaseCustodyRestrictions.MEDIA,
+                        ].includes(restriction),
+                      )
+                      ?.map((custodyRestriction, index) => (
+                        <Box marginTop={index > 0 ? 1 : 0} key={index}>
+                          <Tag
+                            variant={getRestrictionTagVariant(
+                              custodyRestriction,
+                            )}
+                            outlined
```

Comment:
Could you please add a `disabled` prop here, otherwise the component gets a background colour when hovered on.